pkg/minikube/node/cache.go

```diff
@@ -173,7 +173,9 @@ func beginDownloadKicBaseImage(g *errgroup.Group, cc *config.ClusterConfig, down
 					return nil
 				}
 			}
-			klog.Infof("failed to download %s, will try fallback image if available: %v", img, err)
+			infoStr := fmt.Sprintf("failed to download %s, will try fallback image if available: %v", img, err)
+			image.ErrFailedDownloadKICImages = errors.New(infoStr)
```

Comment:
Usually, global `Err*` variables are supposed to be read-only, excluding maybe init functions. The idea is to be able to have checks like `if err == image.ErrFailedDownloadKICImages { ... `.  Changing value of `Err*` makes such checks undetermined and hard to debug.